templates/kubernetes/terraform/modules/kubernetes/db_user/mysql/main.tf

```diff
@@ -0,0 +1,97 @@
+# Create/Update mysql db user with kubernetes job
+resource "random_string" "job_id" {
+  length  = 8
+  lower   = true
+  upper   = false 
+  special = false
+
+  keepers = {
+    password = var.db_app_password
+  }
+}
+
+resource "kubernetes_namespace" "db_ops" {
+  metadata {
+    name = "db-ops"
+  }
+}
+
+resource "kubernetes_secret" "db_create_users" {
+  metadata {
+    name      = "db-create-users"
+    namespace = "db-ops"
+  }
+
+  data = {
+    "create-user.sql" = <<-EOF
+      DROP USER IF EXISTS ${var.db_app_user};
+      CREATE USER ${var.db_app_user} IDENTIFIED BY "${var.db_app_password}";
+      GRANT ALL PRIVILEGES ON ${var.db_name}.* TO ${var.db_app_user};
+EOF
+    "RDS_MASTER_PASSWORD" = var.db_master_password
+  }
+
+  type = "Opaque"
+}
+
+resource "kubernetes_job" "db_create_users" {
```

Comment:
I think we should try to make sure this is only ever run once.. If the dev goes in and changes the privileges or changes something about the user we don't want this to run and wipe their changes. Not an ideal situation for something in terraform..

Reply:
as I conformed, as long as that password version variable doesn't change, that job resource "kubernetes_job" "db_create_users"  has no change, and TF will never run it.